app/templates/index.hbs

```diff
@@ -1,32 +1,14 @@
 <main class="stream index">
   <article class="stream__article">
-    <header class="index__header__wrap">
-      <div class="stream__header index__header">
-        <h1 class="stream__head index__head">{{ t 'index.header_head' }}</h1>
-        <h2 class="stream__subhead">{{ t 'index.header_subhead' }}</h2>
-        <form class="search__wrap">
-          <div class="search__field__wrap">
-            {{input
-              class="search"
-              placeholder=( t 'pageheader.search_placeholder' )
-              value=query
-              enter='query'
-              required='required'
-            }}
-            <button class="search__reset fa fa-close" type="reset"></button>
-          </div>
-          <input class="search__submit btn btn--standalone btn--search" type="submit" value="{{ t 'pageheader.search_link' }}" {{action 'query' query}}>
-        </form>
-      </div>
-    </header>
-    <div class="index__questions__mod">
-      <section class="stream__section index__section index__questions__wrap">
-        <h3 class="section__head">{{ t 'index.questions_head' }}</h3>
-        <p class="section__p">{{ t 'index.questions_subhead' }}</p>
+    <section class="stream__section index__section index__questions__wrap">
+      <div class="stream__section__wrap">
+        <h3 class="section__head">{{ t 'index.questions_head' }}, {{ t 'index.questions_subhead' }}</h3>
```

Comment:
For this case, put it all together as one thing in the translation file, because the comma is part of the copy. 

It's sort of a happy coincidence that in both english and spanish this line can be split into equivalent clauses separated by a comma. (and probably evidence that we're not really localizing the copy enough)